commands/logs.js

```diff
@@ -1,11 +1,20 @@
-import { followLogs, getLatestLogs } from '../lib/log.js'
+import { followLogs, getLatestLogs, parseLog, formatLog } from '../lib/log.js'
 
 export const logs = async ({ module, follow }) => {
   if (follow) {
     for await (const line of followLogs(module)) {
-      console.log(line)
+      const { text, date } = parseLog(line)
+      process.stdout.write(formatLog(text, { date, isHumanReadable: true }))
     }
   } else {
-    process.stdout.write(await getLatestLogs(module))
+    const lines = (await getLatestLogs(module))
+      .toString()
+      .trim()
+      .split('\n')
+      .filter(Boolean)
```

Comment:
Could you please remind me what this filter does? Are you filtering out empty lines?

Reply:
Cool! Would you mind adding a code comment to clarify this to readers unfamiliar with this trick?

Reply:
Instead of using a trick, I would rather have something self explanatory. How would you write this part of the logic?

Reply:
maybe like this?

```js
.filter(line => line !== '')
```

Reply:
I am fine with the trick if it's accompanied by a comment.

Your solution looks good to me too 👍🏻 

> ```js
> .filter(line => line !== '')
>```

Reply:
I don't want to use anything considered a "trick" because that makes the code harder to read, and if we also need to add a comment that makes it even more harder to read. The more explicit solution solves this elegantly.